backend/app/controllers/backend/albums_controller.rb

```diff
@@ -87,7 +87,7 @@ def set_album
       end
 
       def set_albums
-        @albums = Album.albums(@search, @index_items_limit * @page)
+        @albums = Album.albums(params, @index_items_limit * @page)
```

Comment:
let's use safe params for this. by creating a method that does something like:

`params.permit(:featured, :visibility_off)`

Makes it safer.